pkg/virt-controller/services/template.go

```diff
@@ -251,12 +251,21 @@ func (t *templateService) RenderLaunchManifestNoVm(vmi *v1.VirtualMachineInstanc
 	return t.renderLaunchManifest(vmi, nil, true)
 }
 
-func (t *templateService) RenderMigrationManifest(vmi *v1.VirtualMachineInstance, pod *k8sv1.Pod) (*k8sv1.Pod, error) {
+func (t *templateService) RenderMigrationManifest(vmi *v1.VirtualMachineInstance, pod *k8sv1.Pod, migrationConfiguration *v1.MigrationConfiguration) (*k8sv1.Pod, error) {
 	reproducibleImageIDs, err := containerdisk.ExtractImageIDsFromSourcePod(vmi, pod)
 	if err != nil {
 		return nil, fmt.Errorf("can not proceed with the migration when no reproducible image digest can be detected: %v", err)
 	}
-	return t.renderLaunchManifest(vmi, reproducibleImageIDs, false)
+	podManifest, err := t.renderLaunchManifest(vmi, reproducibleImageIDs, false)
+	if err != nil {
+		return nil, err
+	}
+
+	// PostCopy needs the userfaultfd syscall which is restricted in the RuntimeDefault seccomp profile
+	if migrationConfiguration.AllowPostCopy != nil && *migrationConfiguration.AllowPostCopy {
```

Comment:
Again, not general enough. Also I would rather do this at `RenderMigrationManifest` caller side.

Reply:
Feature gate added in new commit